game_clean/copy/start_tf2.bat

```diff
@@ -1,2 +1,2 @@
-@echo off
-hl2.bat -novid -nojoy -nosteamcontroller -nohltv -particle_fallback 2 %* < nul
+@ECHO OFF
+hl2.bat -game tf %* < nul

```

Comment:
start_tf2 is separate, to isolate "required" launch flags with optional ones.

Reply:
Is `-particle_fallback 2` required? (right file this time lol)

Reply:
I added it there for convenience and so people can test the new option easily. It's not necessarily required but I would like to include it.

Reply:
I'll add it back, then. Should I add `-nostartupsound`?

Reply:
No, I'd rather keep the current set of launch options with no changes.